File: C/soma_vetores.c
#include <stdio.h>

int main()
{
    int n;

    printf("Quantos elementos vai ter cada vetor? ");
    scanf("%d", &n);

    int A[n], B[n], C[n];

    printf("Digite os valores do vetor A:\n");
    for(int i = 0; i < n; i++)
    {
        scanf("%d", &A[i]);
    }

    printf("Digite os valores do vetor B:\n");
    for(int i = 0; i < n; i++)
    {
        scanf("%d", &B[i]);
    }

    for(int i = 0; i < n; i++)
    {
        C[i] = A[i] + B[i];
    }

    printf("VETOR RESULTANTE:\n");
    for(int i = 0; i < n; i++)
    {
        printf("%d\n", C[i]);
    }
    return 0;
}

File: Python/soma_vetores.py
n = int(input("Quantos valores vai ter cada vetor? "))

A = [0 for x in range(n)]
B = [0 for x in range(n)]
C = [0 for x in range(n)]

print("Digite os valores do vetor A:")
for i in range(0, n):
    A[i] = int(input())

print("Digite os valores do vetor B:")
for i in range(0, n):
    B[i] = int(input())

for i in range(0, n):
    C[i] = A[i] + B[i]

print("VETOR RESULTANTE:")
for i in range(0, n):
    print(C[i])
